build-support/bin/ci.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+function clone_repo {
+  local name=$1
+  local repo=$2
+  local branch=$3
+  git clone $repo || {
+    echo "Unable to clone $name from $repo"
+    exit 1
+  }
+  cd $name || {
+    echo "Unable to get $name from $repo"
+    exit 2
+  }
+  git checkout $branch || {
+    echo "Unable to checkout branch $branch for $name from $repo"
+    exit 3
+  }
+  cd ..
+}
+
+function publish_local {
+  local name=$1
+  local target=$2
+  cd $name || {
+    echo "Unable to get $name for publish local $target"
+    exit 4
+  }
+  ../../../sbt ++$TRAVIS_SCALA_VERSION $target || {
```

Comment:
using the ./sbt included with finagle and util gives errors, exactly the error is ->

bad sbt-launch.jar.  delete sbt-launch.jar and run ./sbt again.

I tried to delete de sbt-launch.jar just for curiosity but the error remains.

For a moment I though it was a local error on my machine but Travis CI gives the same error.

So, I just use the sbt included on scrooge.

Do you have the same error using ./sbt from finagle ?

I will try to figure out what the error is and we can discuss about it.